package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "standard 'src/**/*.js' 'test/**/*.js' storages.js constants.js && BABEL_ENV=commonjs ava",
     "test:watch": "npm test -- --watch",
-    "copy-types": "cp -r type-definitions/ lib/",
-    "clean": "rimraf lib dist es",
+    "copy-types": "rsync -av --exclude=constants.d.ts --exclude=storages.d.ts type-definitions/ lib/ && cp type-definitions/constants.d.ts type-definitions/storages.d.ts ./",
```

Comment:
I am not sure the module declaration will be deprecated, at lease they will be useful for patching, when some issues in origin one. ([example](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html), working by the [declaration-merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) rule)